docs/template-migration/phases/active-migration/testing-on-tugboat.md

```diff
@@ -0,0 +1,42 @@
+# Testing on Tugboat and other environments
+
+## Using Tugboat for Next Build work
+
+> [!NOTE]
+> On a next-build tugboat, only next-build pages will be visible. You will not be able to access content-build pages; these pages will result in a 404. Therefore, you will not be able to navigate the tugboat in the way you would a typical website, you must go directly to the URL you want to test.
+
+### Next-build Tugboat
+
+Every Next Build PR creates a Tugboat environment using that PR code and runs a static (content release) build on that Tugboat environment. When the static build is complete, you can see what Next Build will look like with your changed code. This Tugboat environment is linked from your Next Build PR, the same way that a CMS Tugboat environment is linked from a va.gov-cms PR.
+
+Note that there are a few limitations to the next-build tugboat instances:
+
+- There are currently CORS errors so they will not show VAMC Facility Real Time Banners
+- The Tugboat also does not show content that relies on the facilities API (e.g. Nearby Vet Center content, Static maps from MapBox, Patient wait times)
```

Comment:
You could say vets-api here. That is where banners and all facilities fetch from.